readme.txt

```diff
@@ -1,5 +1,5 @@
 === Jobber ===
-Contributors: Jobber
+Contributors: 10up
```

Comment:
I'm not sure what the username of the Jobber account is going to be but I would guess we would want that here, maybe in addition to 10up or at least alongside 10up